Makefile

```diff
@@ -39,9 +39,10 @@ GIT_TAG=release-$(CATALOG_VERSION)
 ALLOW_DIRTY_CHECKOUT?=false
 SOURCE_DIR := operators
 
+MANIFESTDIR := ./manifests
 # List of github.org repositories containing operators
 # This is in the format username/reponame separated by space:  user1/repo1 user2/repo2 user3/repo3
-OPERATORS := openshift/dedicated-admin-operator
+OPERATORS := openshift/dedicated-admin-operator dak1n1/configure-alertmanager-operator
```

Comment:
```suggestion
OPERATORS := openshift/dedicated-admin-operator openshift/configure-alertmanager-operator
```

Reply:
If we don't have this in the openshift org it needs created.  This is a requirement of the operator work. FYI @dak1n1

Reply:
Whoops, you're right. The repo exists, but she's still curating her PR.